lbrynet/conf.py

```diff
@@ -1,116 +1,99 @@
+        get_path(FOLDERID.RoamingAppData, UserHandle.current), 'lbryum')
 else:
     platform = LINUX
-    default_download_directory = os.path.join(os.path.expanduser("~"), 'Downloads')
-    default_data_dir = os.path.join(os.path.expanduser("~"), ".lbrynet")
-    default_lbryum_dir = os.path.join(os.path.expanduser("~"), ".lbryum")
-
+    default_download_directory = os.path.join(os.path.expanduser('~'), 'Downloads')
+    default_data_dir = os.path.join(os.path.expanduser('~'), '.lbrynet')
+    default_lbryum_dir = os.path.join(os.path.expanduser('~'), '.lbryum')
 
-ICON_PATH = "icons" if platform is WINDOWS else "app.icns"
+ICON_PATH = 'icons' if platform is WINDOWS else 'app.icns'
 
 
-class Settings(object):
-    """A collection of configuration settings"""
-    __fixed = []
-    _excluded = ['get_dict', 'update']
-
-    def __iter__(self):
-        for k in self.__dict__.iterkeys():
-            if k.startswith('_') or k in self._excluded:
-                continue
-            yield k
-
-    def __getitem__(self, item):
-        assert item in self, IndexError
-        return self.__dict__[item]
-
-    def __setitem__(self, key, value):
-        assert key in self and key not in self.__fixed, KeyError(key)
-        self.__dict__[key] = value
-
-    def __contains__(self, item):
-        return item in iter(self)
-
-    def get_dict(self):
-        return {k: self[k] for k in self}
-
-    def update(self, updated_settings):
-        for k, v in updated_settings.iteritems():
-            try:
-                self.__setitem__(k, v)
-            except (KeyError, AssertionError):
-                pass
+def server_port(server_and_port):
+    server, port = server_and_port.split(':')
+    return server, int(port)
 
 
 class Env(envparse.Env):
     """An Env parser that automatically namespaces the variables with LBRY"""
-    NAMESPACE = 'LBRY_'
+
     def __init__(self, **schema):
         self.original_schema = schema
         my_schema = {
             self._convert_key(key): self._convert_value(value)
             for key, value in schema.items()
-        }
+            }
         envparse.Env.__init__(self, **my_schema)
 
     def __call__(self, key, *args, **kwargs):
         my_key = self._convert_key(key)
         return super(Env, self).__call__(my_key, *args, **kwargs)
 
-    def _convert_key(self, key):
-        return Env.NAMESPACE + key.upper()
-
-    def _convert_value(self, value):
-        """Allow value to be specified as an object, tuple or dict
+    @staticmethod
+    def _convert_key(key):
+        return ENV_NAMESPACE + key.upper()
 
-        if object or dict, follow default envparse rules, if tuple
-        it needs to be of the form (cast, default) or (cast, default, subcast)
+    @staticmethod
+    def _convert_value(value):
+        """
+        Allow value to be specified as a tuple or list. If you do this, the tuple/list
+        must be of the form (cast, default) or (cast, default, subcast)
```

Comment:
No need to change the docstring. The original docstring is inline with PEP257.  Docstrings are supposed to be:

```
"""A single line that provides a short description.

An optional longer explanation after a blank line. With the closing quotes
on a line on their own.
"""
```